src/Kernel/Protocol.class.st

```diff
@@ -28,6 +43,17 @@ Protocol class >> name: nm methodSelectors: methods [
 		yourself
 ]
 
+{ #category : #accessing }
+Protocol class >> nullProtocolName [
+
+	^ #'no messages'
```

Comment:
This one make no sense, it's a presentation thing. Is it even used?

Reply:
It is on my TODO list to check what uses that and probably kill that because I don't like it. 

But it's for another PR :)